charts/akeyless-secure-remote-access/templates/akeyless-zero-trust-portal/deployment.yaml

```diff
@@ -0,0 +1,98 @@
+{{- if .Values.ztpConfig.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ztp-{{ include "akeyless-secure-remote-access.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{- include "akeyless-secure-remote-access.labels" . | nindent 4 }}
+  {{- if .Values.deployment.labels }}
+    {{- toYaml .Values.deployment.labels | nindent 4 }}
+  {{- end }}
+spec:
+  selector:
+    matchLabels:
+      app: zero-trust-portal
+  {{- if not .Values.ztpConfig.HPA.enabled }}
+  replicas: {{ .Values.ztbConfig.replicaCount }}
+  {{- end }}
+  template:
+    metadata:
+      labels:
+        app: zero-trust-portal
+    spec:
+      {{- if .Values.ztpConfig.securityContext }}
+        {{- if .Values.ztpConfig.securityContext.enabled }}
+      securityContext:
+        runAsUser: {{ .Values.ztpConfig.securityContext.runAsUser }}
+        fsGroup: {{ .Values.ztpConfig.securityContext.fsGroup }}
+        {{- end }}
+      {{- end }}
+      serviceAccountName: {{ template "akeyless-zero-trust-portal.getServiceAccountName" . }}
+      {{- if .Values.ztpConfig.nodeSelector }}
+      nodeSelector:
+      {{ toYaml .Values.ztpConfig.nodeSelector | indent 4 }}
+      {{- end }}
+      imagePullSecrets:
+        - name: akeyless-docker-hub
```

Comment:
This secrete is not defined in the customer side - missing.
If the repo is private then you need to create a secrete for it, or a dedicate reference section in the values.yaml file as the other images of SRA here

Reply:
but this secret is in charts/akeyless-secure-remote-access/templates/secrets.yaml